src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -435,14 +435,6 @@
       {:endpoint endpoints/list-workspaces
        :on-done (net/handle-ajax-response
                  #(swap! state update :server-response assoc :workspaces-response %))})
-     (endpoints/get-billing-projects
-      (fn [err-text projects]
-        (if err-text
-          (swap! state update :server-response assoc
-                 :error-message err-text :disabled-reason :error)
-          (swap! state update :server-response assoc
-                 :billing-projects (map :projectName projects)
-                 :disabled-reason (when (empty? projects) :no-billing)))))
```

Comment:
`create/Button` is the component that makes the remaining request for billing projects, right? (since dev is down rn I can't test empirically)

Reply:
Yes